Backpacks.cs

```diff
@@ -686,12 +688,13 @@ public ushort BackpackSize
         private class Backpack
         {
             private bool _initialized = false;
-            private string _ownerIdString;
             private bool _hasPossibleOverflow = false;
 
             private ItemContainer _itemContainer = new ItemContainer();
             private List<BasePlayer> _looters = new List<BasePlayer>();
 
+            public string OwnerIdString { get; private set; }
```

Comment:
I believe this is already ignored by the JSON serialization because of the private setter, but perhaps it might be good to make that more clear by adding an explicit [JsonIgnore] attribute here.

Reply:
Good catch. There was an OwnerIdString in my data file (though it probably didn't hurt).